python/practices/practice-3/src/path1/long_words_printer.py

```diff
@@ -0,0 +1,32 @@
+"""
+Check and print words with more than 20 letters from a word file
+"""
+
+
+import os
+
+
+def print_words(file):
+    """
+    Print words from a file that have a length greater than 20 characters.
+
+    Args:
+        - file (str): The path to the input file containing a list of words.
+    """
+
+    fin = open(file, encoding='utf-8')
```

Comment:
Please create the helper's function to read the file.

Reply:
Done